app/jobs/modsulator_job.rb

```diff
@@ -256,4 +258,14 @@ def status_ok(dor_object)
     status = dor_object.status_info[:status_code]
     [1, 6, 7, 8, 9].include?(status)
   end
+
+  private
+
+  def client
+    Faraday.new do |f|
+      f.request :multipart
+      f.request :url_encoded
+      f.adapter :net_http
```

Comment:
Sure, looking into alternative HTTP adapters makes a lot of sense. Could we do that separate from switching to Faraday in the first place?